test/Libraries/RevitNodesTests/Elements/ElementTests.cs

```diff
@@ -436,5 +436,42 @@ private static void AssertElementsAreJoined(Element element, Element otherElemen
         }
 
         #endregion
+
+        [Test]
+        [TestModel(@".\Element\elementIntersection.rvt")]
+        public void CanSuccessfullyGetIntersectingElementsOfSpeceficCategory()
```

Comment:
same as other PR, don't like `Successfully` in the name - despite existing tests being like this.